### PR TITLE
Feature/worker node scale

### PR DESCRIPTION
**Pull request by waynedovey**

### CODE REVIEW

File: helpernode-build.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    read -r BUILD_LAB_CHOICE
+    if [ "${BUILD_LAB_CHOICE}" != "" ]; then
+        BUILD_LAB=${BUILD_LAB_CHOICE}
+    fi
+fi
+printf "* Cluster Name: ${BUILD_LAB}\n\n"
+
+mkdir -p roles
+git clone https://github.com/waynedovey/ocp4-upi-helpernode.git roles/ocp4-upi-helpernode
+git clone https://github.com/waynedovey/389-ldap-server.git roles/389-ldap-server
```
Comment (luckylittle) [inline helpernode-build.sh]: This should be done using `requirements.yml` that i already created for this:
`./install_requirements.sh`

Comment (waynedovey) [reply helpernode-build.sh]: Cool! THanks

File: scale-down-nodes-vsphere.yml [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+      loop:
+        - { name : "worker3" }
+        - { name : "worker4" }
+        - { name : "worker5" }
+      ignore_errors: yes
+      tags:
+        - "1"
+
+    - name: OCPNODES 1.1 | Drain Nodes
+      shell: oc adm drain "{{ item.name }}.{{ dns.clusterid }}.{{ dns.domain }}" --ignore-daemonsets
```
Comment (luckylittle) [inline scale-down-nodes-vsphere.yml]: Add `--delete-local-data` here?

File: scale-down-nodes-vsphere.yml [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+
+  environment:
+    # KUBECONFIG: ./kubeconfig
+    # KUBECONFIG: /root/.kube/config
+    KUBECONFIG: /root/ocp4-vsphere-upi-automation/install-dir/auth/kubeconfig
+
+  tasks:
+    - name: OCPNODES 1.0 | Cordon Nodes
+      shell: oc adm cordon "{{ item.name }}.{{ dns.clusterid }}.{{ dns.domain }}"
```
Comment (luckylittle) [inline scale-down-nodes-vsphere.yml:18]: Shouldn't this also include `oc adm manage-node --schedulable=false <node>`?

**Review by luckylittle — APPROVED** [review]
Pretty sweet

File: scale-down-nodes-vsphere.yml (hunk 2 above)
Comment (luckylittle) [reply scale-down-nodes-vsphere.yml]: Maybe not needed when you actually completely delete them anyway...

**Review by luckylittle — APPROVED** [review]

File: scale-down-nodes-vsphere.yml (hunk 2 above)
Comment (waynedovey) [reply scale-down-nodes-vsphere.yml]: I think it's required if there is state, I will include. Thanks!

File: scale-down-nodes-vsphere.yml (hunk 3 above)
Comment (waynedovey) [reply scale-down-nodes-vsphere.yml:18]: Cordon is the same thing. Shouldn't be required.

**Pull request merged**